docs/_docs/2_3_querying_data.md

```diff
@@ -84,55 +102,53 @@ using the hive session property for incremental queries: `set hive.fetch.task.co
 would ensure Map Reduce execution is chosen for a Hive query, which combines partitions (comma
 separated) and calls InputFormat.listStatus() only once with all those partitions.
 
-## Spark
+## Spark datasource
```

Comment:
For this section, can you take another stab.. it feels short and curt.. may be set some context on things like : Spark Datasources directly query underlying DFS data without Hive (and for this reason I think we should move `SparkSQL` up and place before this section)

Reply:
+1

Reply:
Agreed. I moved this section below Spark SQL. I need some help here with adding additional context though.